cmd/gapit/export_replay.go

```diff
@@ -210,7 +210,7 @@ func (verb *exportReplayVerb) Run(ctx context.Context, flags flag.FlagSet) error
 			sdkPath = os.ExpandEnv("${ANDROID_SDK_HOME}")
 		}
 		if _, err := os.Stat(sdkPath); err != nil {
-			return err
+			return log.Err(ctx, err, "Cannot find Android SDK. Please set ANDROID_SDK_HOME, or use -sdkpath flag")
```

Comment:
use the -skpath flag*

Other than the minor grammar suggestion LGTM!

Reply:
Done, good catch, thanks!